src/modules/storage/LedgerStorage.ts

```diff
@@ -335,6 +342,54 @@ export class LedgerStorage extends Storages {
             PRIMARY KEY(time_stamp, granularity(64))
         );
 
+        CREATE TABLE IF NOT EXISTS proposal_fee
+        (
+            proposal_id          TEXT      NOT NULL,
+            block_height         INTEGER   NOT NULL,
+            tx_hash              TINYBLOB  NOT NULL,
+
+            PRIMARY KEY(proposal_id(64), block_height, tx_hash(64))
+        );
+
+        CREATE TABLE IF NOT EXISTS proposal
+        (
+            proposal_id              TEXT        NOT NULL,
+            block_height             INTEGER     NOT NULL,
+            tx_hash                  TINYBLOB    NOT NULL,
+            app_name                 TEXT        NOT NULL,
+            proposal_type            INTEGER     NOT NULL,
+            proposal_title           TEXT        NOT NULL,
+            vote_start_height        INTEGER     NOT NULL,
+            vote_end_height          INTEGER     NOT NULL,
+            doc_hash                 TINYBLOB    NOT NULL,
+            fund_amount              BIGINT(20)  NOT NULL,
+            proposal_fee             BIGINT(20)  NOT NULL,
+            vote_fee                 BIGINT(20)  NOT NULL,
+            tx_hash_proposal_fee     TINYBLOB    NOT NULL,
+            proposer_address         TEXT        NOT NULL,
+            proposal_fee_address     TEXT        NOT NULL,
+            proposal_status          TEXT        NOT NULL,
+            data_collection_status   TEXT        NOT NULL,
+
+            PRIMARY KEY(block_height, proposal_id(64), app_name(64))
+        );
+        
+        CREATE TABLE IF NOT EXISTS proposal_metadata
+        (
+            proposal_id               TEXT      NOT NULL,
+            voting_start_date         INTEGER   NOT NULL,
+            voting_end_date           INTEGER   NOT NULL,
+            voting_fee_hash           TINYBLOB  NOT NULL,
+            detail                    TEXT      NOT NULL,
+            submit_time               INTEGER   NOT NULL,
+            ave_pre_evaluation_score  INTEGER   NOT NULL,
+            pre_evaluation_start_time INTEGER   NOT NULL,
+            pre_evaluation_end_time   INTEGER   NOT NULL,
+            proposer_name             TEXT,
+
```

Comment:
The attachment file is in the form of an array.
doc_url...
There are also several pre-evaluation information items.
I have discussed this with @ahmedrns  before.